myhoard/restore_coordinator.py

```diff
@@ -408,6 +408,7 @@ def apply_binlogs(self):
                 self.log.info("No GTID ranges found in last file with given target timestamp, finalizing restore")
                 all_gtids_applied = True
 
+        self._purge_old_slave_data()
```

Comment:
This should only be done the very first time this function is called.

Reply:
Oops.